README.md

```diff
@@ -41,7 +41,8 @@ This operator ships two CRDs(the `Build` and `BuildRun`) in order to register a
 Examples of `Build` resource using the example strategies shipped with this operator.
 
 * [`buildah`](samples/build/build_buildah_cr.yaml)
-* [`buildpacks-v3`](samples/build/build_buildpacks-v3_cr.yaml)
+* [`buildpacks-v3-heroku`](samples/build/build_buildpacks-v3-heroku_cr.yaml)
+* [`buildpacks-v3-paketo`](samples/build/build_buildpacks-v3-paketo_cr.yaml)
```

Comment:
it should be `buildpacks-v3` and not `buildpacks-v3-paketo`. As in the PR discussion, whenever we speak about `buildpacks-v3` we all imply `paketo` for now. Please change all instances that include paketo, to only say `buildpacks-v3`

Reply:
Also the files names.

Reply:
@qu1queee ,Thanks for your review! Have modified all `buildpack-v3-paketo` to `buildpack-v3`.